src/base/perf_monitor.c

```diff
@@ -2238,9 +2245,21 @@ perfmon_server_dump_stats_to_buffer (const UINT64 * stats, char *buffer, int buf
 	    {
 	      if (pstat_Metadata[i].valtype != PSTAT_COUNTER_TIMER_VALUE)
```

Comment:
Please consider to change the line if you include all peeked type stats. 
I prefer to something like this:
```c
if ( ... == PSTAT_PEEK_SINGLE_VALUE)
  {
    ...
  }
else if ( != PSTAT_COUNTER_TIMER_VALUE)
  {
    ...
  }
```

Then, you don't need `get_value_from_stat`.